test/functional/pdo_sqlsrv/pdo_fetch_cursorBuffered_decimal_bindColumn_int.phpt

```diff
@@ -1,91 +1,96 @@
-$stmt = $conn->prepare($query);
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch(PDO::FETCH_BOUND);
-var_dump ($decimal_col);
+    $query = "INSERT INTO $tbname VALUES(:p0)";
+    $stmt = $conn->prepare($query);
+    $stmt->bindValue(':p0', $sample, PDO::PARAM_INT);
+    $stmt->execute();
 
-print "\nno buffered cursor, stringify on, fetch_numeric on\n";
-$conn->setAttribute( PDO::ATTR_STRINGIFY_FETCHES, true);
-$stmt = $conn->prepare($query);
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch(PDO::FETCH_BOUND);
-var_dump ($decimal_col);
+    $query = "SELECT exist FROM $tbname";
 
-print "\nno buffered cursor, stringify on, fetch_numeric off\n";
-$conn->setAttribute( PDO::SQLSRV_ATTR_FETCHES_NUMERIC_TYPE, false);
+    //prepare with no buffered cursor
+print "no buffered cursor, stringify off, fetch_numeric off\n"; //stringify and fetch_numeric is off by default
 $stmt = $conn->prepare($query);
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch(PDO::FETCH_BOUND);
-var_dump ($decimal_col);
-
-//prepare with client buffered cursor
-print "\nbuffered cursor, stringify off, fetch_numeric off\n";
-$conn->setAttribute( PDO::ATTR_STRINGIFY_FETCHES, false);
-$stmt = $conn->prepare($query, array(PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE => PDO::SQLSRV_CURSOR_BUFFERED));
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch(PDO::FETCH_BOUND);
-var_dump ($decimal_col);
-
-print "\nbuffered cursor, stringify off, fetch_numeric on\n";
-$conn->setAttribute( PDO::SQLSRV_ATTR_FETCHES_NUMERIC_TYPE, true);
-$stmt = $conn->prepare($query, array(PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE => PDO::SQLSRV_CURSOR_BUFFERED));
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch(PDO::FETCH_BOUND);
-var_dump ($decimal_col);
-
-print "\nbuffered cursor, stringify on, fetch_numeric on\n";
-$conn->setAttribute( PDO::ATTR_STRINGIFY_FETCHES, true);
-$stmt = $conn->prepare($query, array(PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE => PDO::SQLSRV_CURSOR_BUFFERED));
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch(PDO::FETCH_BOUND);
-var_dump ($decimal_col);
-
-print "\nbuffered cursor, stringify on, fetch_numeric off\n";
-$conn->setAttribute( PDO::SQLSRV_ATTR_FETCHES_NUMERIC_TYPE, false);
-$stmt = $conn->prepare($query, array(PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE => PDO::SQLSRV_CURSOR_BUFFERED));
-$stmt->execute();
-$stmt->bindColumn('exist', $decimal_col, PDO::PARAM_INT);
-$value = $stmt->fetch(PDO::FETCH_BOUND);
-var_dump ($decimal_col);
-
-$stmt = null;
-$conn = null;
-
+    $stmt->execute();
```

Comment:
indentation? same for pdo_fetch_cursorBuffered_float.phpt and a few other similar tests

Reply:
ok